osmoutils/accum/accum.go

```diff
@@ -98,7 +98,7 @@ func (accum AccumulatorObject) GetPosition(name string) (Record, error) {
 	return position, nil
 }
 
-func setAccumulator(accum AccumulatorObject, value sdk.DecCoins, shares sdk.Dec) {
+func setAccumulator(accum *AccumulatorObject, value sdk.DecCoins, shares sdk.Dec) {
```

Comment:
what's the reason we need to change the argument to pointer? Seems like here we might not need to change the argument

Reply:
no on the second thought, I think this change makes sense given that all the callers have accum as pointers anyways